docs/dev/android/sdk-ndk-versions.md

```diff
@@ -3,12 +3,12 @@
 The Glean SDK implementation is currently build against the following versions:
 
 * SDK API 34
```

Comment:
```suggestion
* SDK API 35
```

Reply:
Might just remove the version number from the top-level item here and for the ndk to cut down on the places it needs changed